### ISSUE TITLE
mark deprecated because of polymer-cli

### ISSUE THREAD

**Issue by MetaMemoryT** [PolymerLabs/polyserve#100]
polymer serve

**Comment by TimvdLippe** [comment]
`polymer-cli` uses this package internally, so it is actually not deprecated.

**Comment by FredKSchott** [comment]
@TimvdLippe is correct, polymer-cli uses `polymer serve` internally. But I agree that the CLI should be recommended in the README (/w the difference explained).

**Comment by justinfagnani** [comment]
I wonder if we want to remove the CLI interface from polyserve altogether

**Comment by FredKSchott** [comment]
+1 from me, no reason for support the same interface in two places

**Comment by TimvdLippe** [comment]
I am actually still using polyserve in applications not generated by `polymer-cli` and I use it while testing patches on the core library. If you were to deprecate the CLI interface here (which I understand ofc), could you make sure that `polymer serve` can work just like I did with polyserve on for example the core library?

**Comment by justinfagnani** [comment]
@TimvdLippe yes, `polymer serve` works just like `polyserve`. The only difference is the existence of the other commands, and the large installation size. That may be enough reason to keep it.

**Comment by FredKSchott** [comment]
+1 for mentioning the CLI in the README for now (similar to https://github.com/Polymer/polymer-build#relationship-to-polymer-cli) and then revisiting polyserve's cli interface once polymer-cli reaches v1.0.
